### DISCUSSION TITLE
Embed resources breaks absolute position of SVG in revealjs?

### DISCUSSION THREAD

**Discussion by jmgirard** [quarto-dev/quarto-cli#10613] [Q&A] [answered]
### Description

Quarto 1.5.56
Windows 11

Works as expected:
````qmd
---
format: revealjs
---

## Quarto

![](https://quarto.org/docs/presentations/revealjs/demo/mini/images/kitten-450-250.jpeg){.absolute top=50 right=50 width="450" height="250"}

![](https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/410.svg){.absolute top=50 right=50 width="450" height="250"}
````

![image](https://github.com/user-attachments/assets/044b8f8f-74a4-4148-aa86-de84e1472641)

SVG in wrong position:
````qmd
---
format: 
  revealjs:
    embed-resources: true
---

## Quarto

![](https://quarto.org/docs/presentations/revealjs/demo/mini/images/kitten-450-250.jpeg){.absolute top=50 right=50 width="450" height="250"}

![](https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/410.svg){.absolute top=50 right=50 width="450" height="250"}
````

![image](https://github.com/user-attachments/assets/6cf267ca-a39f-4505-bb29-4337a0cdf284)

**Comment by mcanouil** [comment, marked as answer]
That's related to change in Pandoc which had other bad side effects:

- https://github.com/quarto-dev/quarto-cli/issues/9803

cc @cscheid @cderv I let you guys convert this to an issue.

**Reply by cderv** [reply]
Yes this is exactly the same problem where the class is not correctly merged on the svg node, but lost.

**Reply by cderv** [reply]
I have added in the existing issue as this is the same.
